Source/WebCore/accessibility/AXCoreObject.cpp

```diff
@@ -634,7 +634,7 @@ AXCoreObject::AccessibilityChildrenVector AXCoreObject::selectedChildren()
     case AccessibilityRole::TreeGrid:
         return selectedRows();
     case AccessibilityRole::TabList:
-        if (auto* selectedTab = selectedTabItem())
+        if (RefPtr selectedTab = selectedTabItem())
             return { { *selectedTab } };
```

Comment:
Just so I understand...why does this need to be a smart pointer? All we do is deref it and put it into a vector. Is it that the static analyzer considers the act of constructing a vector non-trivial, thus being able to theoretically invalidate the raw pointer?

Also, `AccessibilityChildrenVector` is `Vector<Ref<AXCoreObject>>`. Thus, if we do in fact actually need a `RefPtr` here, I think we can save ref-count churn by using `RefPtr::releaseNonNull` here rather than `*selectedTab`.